distributed/worker.py

```diff
@@ -4576,3 +4591,74 @@ def warn(*args, **kwargs):
         worker.log_event("warn", {"args": args, "kwargs": kwargs})
 
     warnings.warn(*args, **kwargs)
+
+
+def benchmark_disk(
+    sizes=["1 kiB", "100 kiB", "1 MiB", "10 MiB", "100 MiB"],
+    rootdir=None,
+    duration=1.0,
+) -> dict:
+    duration = parse_timedelta(duration)
```

Comment:
why bother? duration can't come from the client and is used exclusively in unit tests

Reply:
Just habit I suppose.  I find passing around strings is nicer.  At some point I wouldn't be sad to see this be more configurable.